tests/integration/aisystems/test_aisystems_tracing.py

```diff
@@ -512,7 +554,9 @@ def parent2(x):
         span_data = [(s.name, s.inputs, s.outputs) for trace in res.data for s in trace.spans]
 
         assert [trace.name for trace in res.data] == ["parent"]
-        assert sorted(span_data) == sorted([("parent", {'x':1, 'y': 2}, 3), ("unit_1", {'x':1}, 1), ("unit_2", {'x':2}, 2)])
+        assert len(span_data) == 3
+        assert sorted(span_data, key=lambda x: x[0]) == sorted([("parent", {'x':1, 'y': 2}, 3), \
+                ("unit_1", {'x':1}, 1), ("unit_2", {'x':2}, 2)], key=lambda x: x[0])
```

Comment:
This was failing because python cannot sort on dictionaries. Maybe this passed because I was accidentally not using python 3 to test when i wrote this? not sure